client/src/Features/MicrosoftLearn/MicrosoftLearnPage.tsx

```diff
@@ -33,6 +35,13 @@ const MicrosoftLearnPageInner = ({ styles }: IStylesOnly<MicrosoftLearnPageStyle
     }
   }, [learnStore, qsParams]);
 
+  useEffect(() => {

```

Comment:
Let's move the store updates to only happen via URI changes and not via any user action. That would essentially help us with a unidirectional data flow and helps reason the code in a better way.

Reply:
This would be taken up as a separate change since moving it to above pattern causes the component to re-render and making it lose focus from the search bar.